e2e/scripts/generate-project.js

```diff
@@ -9,6 +9,13 @@ const config = require("../config.js");
 const { program, Argument } = require("commander");
 const { mkdirIfNotExists } = require("./utils.js");
 
+const GENERATE_WITH_PRESET = [
+  {
+    projectKey: "retail-app-private-client",
+    presetId: "retail-react-app-private-slas-client",
+  },
+];
```

Comment:
I was a bit curious why we have this here rather than in the config file. I guess the contents in the config file handle generating the public client projects while here we call the preset for private client?

Reply:
Could we simplify this script a bit if projectKey === presetId?

Reply:
Moved presets to config